test/Recollateralization.test.ts

```diff
@@ -3088,6 +3113,141 @@ describe(`Recollateralization - P${IMPLEMENTATION}`, () => {
+            await advanceToTimestamp((await getLatestBlockTimestamp()) + 12)
+          }
+        })
+
+        it('Should handle no bid case correctly', async () => {
+          await backingManager.rebalance(TradeKind.DUTCH_AUCTION)
+          const trade = await ethers.getContractAt(
+            'DutchTrade',
+            await backingManager.trades(token0.address)
+          )
+          await token1.connect(addr1).approve(trade.address, initialBal)
+          await advanceToTimestamp((await getLatestBlockTimestamp()) + auctionLength - 1)
+          await expect(
+            trade.connect(addr1).bidAmount(await getLatestBlockTimestamp())
+          ).to.be.revertedWith('auction over')
+          await expect(trade.connect(addr1).bid()).be.revertedWith('auction over')
+
+          // Should be able to settle
+          await expect(trade.settle()).to.be.revertedWith('only origin can settle')
+          await expect(backingManager.settleTrade(token0.address))
+            .to.emit(backingManager, 'TradeSettled')
+            .withArgs(trade.address, token0.address, token1.address, 0, 0)
+
+          // Should NOT start another auction, since caller was not DutchTrade
+          expect(await backingManager.tradesOpen()).to.equal(0)
+        })
+
+        it('Should bid in final second of auction and launch another auction', async () => {
+          await backingManager.rebalance(TradeKind.DUTCH_AUCTION)
+          const trade = await ethers.getContractAt(
+            'DutchTrade',
+            await backingManager.trades(token0.address)
+          )
+          await token1.connect(addr1).approve(trade.address, initialBal)
+
+          // Snipe auction at 1s left
+          await advanceToTimestamp((await getLatestBlockTimestamp()) + auctionLength - 3)
+          await trade.connect(addr1).bid()
+          expect(await trade.canSettle()).to.equal(false)
+          expect(await trade.status()).to.equal(2) // Status.CLOSED
+          expect(await trade.bidder()).to.equal(addr1.address)
+          expect(await token0.balanceOf(addr1.address)).to.equal(initialBal)
+
+          const expected = divCeil(
+            await dutchBuyAmount(
+              fp('299').div(300), // after all txs in this test, will be left at 299/300s
+              collateral0.address,
+              collateral1.address,
+              issueAmount,
+              config.minTradeVolume,
+              config.maxTradeSlippage
+            ),
+            bn('1e12') // decimals
+          )
+          expect(await backingManager.tradesOpen()).to.equal(1)
+          expect(await token1.balanceOf(backingManager.address)).to.equal(expected)
+
+          // Should launch another auction: RSR -> token1
+          expect(await backingManager.trades(token0.address)).to.equal(ZERO_ADDRESS)
+          expect(await backingManager.trades(rsr.address)).to.not.equal(ZERO_ADDRESS)
+          expect(await backingManager.tradesOpen()).to.equal(1)
+        })
+      })
```

Comment:
probably worth testing a couple more scenarios:
1) token dutch auction, rsr dutch auction, complete auction and check full recollateralization
2) token dutch auction, timeout rsr dutch auction and send it to a batch auction, complete and check full recollateralization

honestly you could probably just extend the above test to cover (1)